apps/functions-subscription/src/domain/users.ts

```diff
@@ -25,4 +25,8 @@ type TrialSubscriber = BaseUser & {
   readonly type: 'subscriber';
 };
 
-export type Tenant = TrialOwner | TrialSubscriber;
+type TrialSupport = BaseUser & {
+  readonly type: 'support';
+};
+
+export type Tenant = TrialOwner | TrialSubscriber | TrialSupport;
```

Comment:
```suggestion
export type Tenant = TrialOwner | TrialSubscriber | Support;
```

**suggestion**: I would not add the `Trial` prefix, because the support user is not related to any trial

Reply:
if you accept this suggestion, please be consistent with other name references